specs/card.proto

```diff
@@ -58,6 +59,11 @@ message CardInfo {
   optional CardExtensions extensions = 3;
 }
 
+message CardVerification {
+  optional bool cardValid = 1;
+  optional uint32 verificationTimeStamp = 2;
```

Comment:
I think we should store it as secondsSinceEpoch (as this is the more-or-less default to store time stamps). And we don't have to think about how we round to days.

```suggestion
  
  optional uint64 verification = 2;
```

Reply:
@michael-markl but the need a separate package for dart? Or how to deal with `Int64` ?
https://pub.dev/packages/fixnum

Reply:
I think this dependency is Okay, it's at least from flutter.dev